src/lib/components/Swap/Output.tsx

```diff
@@ -31,6 +32,30 @@ const OutputColumn = styled(Column)<{ hasColor: boolean | null }>`
   }
 `
 
+const UniswapA = styled.a`
+  align-items: center;
```

Comment:
I don't think these addition properties (like `align-items`) are necessary if you use a `Row`.